examples/examples.py

```diff
@@ -100,8 +102,8 @@
     - local_db: without database and grafana
     - timescale: with database and grafana (note: you need docker installed)
     """
-    data_format = "local_db"  # "local_db" or "timescale"
-    example = "small"
+    data_format = "timescale"  # "local_db" or "timescale"
+    example = "small_learning_storage"
```

Comment:
please do not adjust the default values of the examples file

Reply:
Nick wanted to continue working on both startegies, so I pushed the example I used, but yes indeed it should be changed back!